fn/chocolatey/tools/chocolateyinstall.ps1

```diff
@@ -0,0 +1,24 @@
+﻿$ErrorActionPreference = 'Stop';
+
+$packageName= 'functions'
+$toolsDir   = "$(Split-Path -parent $MyInvocation.MyCommand.Definition)"
+$url        = 'https://github.com/iron-io/functions/releases/download/0.2.25/fn.exe'
```

Comment:
Yes, we need to make the scripts update the version on release, same way as we do for git, docker, cli install, etc. 

How does this get submitted to Chocolatey?  We'll want to make sure that's automated too.